sway-fmt-v2/src/items/item_enum.rs

```diff
@@ -63,7 +63,7 @@ impl Format for ItemEnum {
                 });
 
                 let mut value_pairs_iter = value_pairs.iter().enumerate().peekable();
-                for (var_index, (type_field, comma_token)) in value_pairs_iter.clone() {
+                while let some((var_index, (type_field, comma_token))) = value_pairs_iter.next() {
```

Comment:
```suggestion
                while let Some((var_index, (type_field, comma_token))) = value_pairs_iter.next() {
```

I think this is why `clippy` and other CI steps are failing. If you could change occurrences of `some(..)` to `Some(..)`, it should be good to go!